packages/wallet/src/components/kyc/VerifyIdentityForm.tsx

```diff
@@ -16,27 +17,42 @@ import { useKYCFormContext } from './context'
 export const VerifyIdentityForm = () => {
   const [openDialog, closeDialog] = useDialog()
   const { idTypes } = useKYCFormContext()
-  const [frontIDFile, setFrontIDFile] = useState('')
-  const [selfieFile, setSelfieFile] = useState('')
-  const [backIDFile, setBackIDFile] = useState('')
   const [isBackRequired, setIsBackRequired] = useState(false)
   const router = useRouter()
 
   const handleFileOnChange = (event: SyntheticEvent<HTMLInputElement>) => {
+    const reader = new FileReader()
     const target = event.currentTarget
-    const fileName = target.value.slice(target.value.lastIndexOf('\\') + 1)
-    if (target.name === 'faceImage') {
-      setSelfieFile(fileName)
-    } else if (target.name === 'frontSideImage') {
-      setFrontIDFile(fileName)
-    } else {
-      setBackIDFile(fileName)
+    if (target.files && target.files.length > 0) {
+      const file = target.files[0]
+      reader.readAsDataURL(file)
+      reader.onloadend = () => {
+        let fileBase64 = reader.result?.toString() || ''
+        fileBase64 = fileBase64.slice(fileBase64.indexOf('base64') + 7)
```

Comment:
From Rapyd docs:
> ERROR_ID_VERIFICATION_BACK_SIDE_IMAGE_MIME_TYPE_INVALID
The request tried to validate identity, but the back side image provided was not the correct MIME type. The request was rejected. Corrective action: Use a JPEG or PNG image.


It looks like only JPEG or PNG images are allowed (from what I'm seeing this applies to all images, not only `back_side_image` - [reference](https://docs.rapyd.net/build-with-rapyd/reference/identity-verification#identity-verification-errors)). We can add to `FileUpload` component `accepts="image/jpeg, image/png"`, but we will still have to do some extra validation (client and server side). We can create a separate issue for this and address it later on.